sdk/cosmos/azure-cosmos/azure/cosmos/offer.py

```diff
@@ -27,10 +27,14 @@ class ThroughputProperties(object):
     To read and update throughput properties use the associated methods on the :class:`Container`.
```

Comment:
Looks like we don't have a proper docstring for this object.
Now that we are expecting people to actually import and use this as input - we will need to fully document the constructor, and all the parameters and how they are used together.
For example - the increment_percent will be ignored if auto_scale_max_throughput is set, and if autoscale is set, I presume offer_througput will be ignored?
These combinations are all pretty confusing - so if the intention is to combine this object - it needs to be really clear how it's used.